Add rule regarding ball play on the nearest neutral spot (#104)
robocup-junior/soccer-rules: rules.adoc

Other players must be able to access the ball.

The ball needs to stay within the bounds of the field, as defined by the
walls. If a robot moves the ball outside of the field (that is, beyond the walls
or above their height), it is deemed damaged. (<<damaged-robots>>)

{~~A robot must touch the ball that is placed no further than 20 cm from any point
on its convex hull within 10 seconds. If a robot does not do so within the time
limit, it is deemed to be damaged. ~>
Any robot must approach and touch the ball when it is placed on the nearest
neutral spot before lack of progress is called. When on its own side of the field,

> any robot must be able to move the ball from the nearest neutral spot to the

Reviewer: What if the neutral spot is on the opponent's side?

> opponent's side of the field. If a robot does not do these things, it is deemed to be damaged.

Reviewer: Which robot is deemed to be damaged? All on the field? The nearest one?

Author: That's going to be the one that is not able to play the ball on the nearest neutral spot. 🙂

walls. If a robot moves the ball outside of the field (that is, beyond the walls
or above their height), it is deemed damaged. (<<damaged-robots>>)

{~~A robot must touch the ball that is placed no further than 20 cm from any point
on its convex hull within 10 seconds. If a robot does not do so within the time
limit, it is deemed to be damaged.
~>Any robot must approach and touch the ball when it is placed on the nearest
neutral spot before lack of progress is called. When on its own side of the field,
any robot must be able to move the ball from the nearest neutral spot to the

> opponent's side of the field. If a robot does not do these things, it is deemed to be damaged.

Reviewer 2: ```suggestion
opponent's side of the field. If a specific robot does act this way, it is deemed to be damaged.
```

Reviewer: `Does not` act or `does act`?
